lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,224 @@
+        finishButton.setTitleTextAttributes([NSAttributedStringKey.font: LucienTheme.Fonts.Muli.semiBold.getFont(size: 17)], for: .normal)
+        finishButton.tintColor = LucienTheme.finishButtonGrey
+        finishButton.isEnabled = false
+        navigationItem.rightBarButtonItem = finishButton
+    }
+
+    private func configureViewController() {
+        // Cover Photo Cell
+        configureAddCoverUIButton(button: coverPhotoButton)
+
+        // Comic Title
+        configureTextFieldBorder(textField: comicTitleTextField)
+        comicTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Comic Title")
+        comicTitleTextField.addTarget(self, action: #selector(AddComicViewController.comicTitleEditingChanged), for: .editingChanged)
+
+        // Story Title
+        configureTextFieldBorder(textField: storyTitleTextField)
+        storyTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Story Title")
+
+        // Volume
+        configureTextFieldBorder(textField: volumeTextField)
+        volumeTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Volume Number")
+
+        // Issue
+        configureTextFieldBorder(textField: issueTextField)
+        issueTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Issue Number")
+
+        // Publisher
+        configureTextFieldBorder(textField: publisherTextField)
+        publisherTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Marvel, DC Comics, Image, Dark Horse…")
+
+        // Release Date
+        configurePickerUIButton(button: selectADateButton)
+
+        // Genre
+        configurePickerUIButton(button: selectAGenreButton)
+
+        // Condition
+        configurePickerUIButton(button: selectAConditionButton)
+    }
+
+    private func configureTextFieldBorder(textField: UITextField) {
+        let bottomBorder = CALayer()
+        bottomBorder.frame = CGRect(x: 0.0, y: textField.frame.height - 3.0, width: textField.frame.width, height: 1.0)
+        bottomBorder.backgroundColor = LucienTheme.silver.cgColor
+        textField.borderStyle = UITextBorderStyle.none
+        textField.layer.addSublayer(bottomBorder)
+    }
+
+    private func configurePickerUIButton(button: UIButton) {
+        button.setImage(UIImage(named: "dropDownArrow"), for: .normal)
+        button.semanticContentAttribute = .forceRightToLeft
+        button.imageEdgeInsets.left = button.frame.width - 24
+        button.tintColor = LucienTheme.dark
+    }
+
+    private func configureAddCoverUIButton(button: UIButton) {
+        button.titleLabel?.lineBreakMode = .byWordWrapping
+        button.titleLabel?.textAlignment = .center
+        button.setTitle("Take \n Cover \n Photo", for: .normal)
+        button.titleLabel?.font = LucienTheme.Fonts.Muli.bold.getFont(size: 13)
```

Comment:
Your fonts aren't showing up in Interface Builder because they're not installed on your computer itself. If you need help with that, let me know